k2/python/k2/__init__.py

```diff
@@ -17,6 +17,8 @@
 from .fsa_properties import to_str as properties_to_str
 from .ops import index
 from .ops import index_add
+from .ragged_shape import RaggedShape
```

Comment:
Please run
```bash
cd k2
flakek8 k2/python
```
and fix the style issues reported by `flake8`.

Reply:
fixed.